.github/workflows/node.js.yml

```diff
@@ -83,3 +96,15 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           directory: ./coverage
           flags: unit
+
+      - name: Install Playwright dependencies
+        run: PLAYWRIGHT_BROWSERS_PATH=0 npx playwright install --with-deps
+      - name: Run Playwright tests
+        run: PLAYWRIGHT_BROWSERS_PATH=0 npx playwright test
+      - name: Upload Playwright report
+        uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: playwright-report
+          path: playwright-report/
+          retention-days: 14
```

Comment:
Any reason to not just leave this as the default maximum of 90 days?

Reply:
90 seems excessive, the Playwright yml defaulted to 30 and I knocked it down to 14. I guess there's no reason not to though, so I've bumped it up to 90.